Command/SwarrotCommand.php

```diff
@@ -49,17 +49,31 @@ public function __construct(
      */
     protected function configure()
     {
+        $defaultPollInterval = isset($this->extras['poll_interval']) ? $this->extras['poll_interval'] : 500000;
```

Comment:
$this->extras['poll_interval'] ?? 500000

Reply:
I copy / pasted existing code but feel free to make a PR. ;)